### PR TITLE
feat(component): add search input component

### PR DESCRIPTION
**Pull request by kduret**
image
focused :
image

### CODE REVIEW

File: src/InputField/SearchInput/index.jsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+import SearchIcon from '@material-ui/icons/Search';
+
+const useAdornmentStyles = makeStyles((theme) => ({
+  searchIcon: {
+    color: 'darkgray',
+    '& > input$focused': {
+      color: theme.palette.primary.main,
+    },
+  },
+  focused: {},
```
Comment (jdelpierre) [inline src/InputField/SearchInput/index.jsx]: Why "focused" is empty, Is necessary to declare it ?

File: src/InputField/SearchInput/index.jsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    </InputAdornment>
+  );
+}
+
+const useStyles = makeStyles((theme) => {
+  // took it from material-ui source code
+  const light = theme.palette.type === 'light';
+  const bottomLineColor = light
+    ? 'rgba(0, 0, 0, 0.42)'
+    : 'rgba(255, 255, 255, 0.7)';
```
Comment (jdelpierre) [inline src/InputField/SearchInput/index.jsx]: the "rgba" is not defined in global theme ? 
If we add another field, we will have to redefine these colors?

File: src/InputField/SearchInput/index.jsx (hunk 1 above)
Comment (kduret) [reply src/InputField/SearchInput/index.jsx]: it is mandatory to link it to searchIcon property ('& > input$focused)

File: src/InputField/SearchInput/index.jsx (hunk 2 above)
Comment (kduret) [reply src/InputField/SearchInput/index.jsx]: I wrote in the comment "took it from material-ui source code" ;)
So, event material-ui does not use it as theme :D

File: src/InputField/SearchInput/index.jsx (hunk 1 above)
Comment (jdelpierre) [reply src/InputField/SearchInput/index.jsx]: Ok ;)

File: src/InputField/SearchInput/index.jsx (hunk 2 above)
Comment (jdelpierre) [reply src/InputField/SearchInput/index.jsx]: Ah okay sorry I didn't look at the associated commentary properly :(

**Pull request merged**